libs/ng-strapi-auth/src/lib/core/services/auth/auth.service.ts

```diff
@@ -38,50 +38,43 @@ export class AuthService {
     }
```

Comment:
no need for the take(1) for getUserFromServer() since it's already in the function.